drp.py

```diff
@@ -256,6 +264,22 @@ def handle_error(exc, retry=True):
         sublime.set_timeout_async(connect_background, 0)
 
 
+def get_git_url(window):
+    for folder in window.folders():
+        gitcfg_path = folder+"/.git/config"
+
+        if (os.path.exists(gitcfg_path)):
+            f = open(gitcfg_path, "r")
+            filteredConfig = ''.join(f.read().split())
+            ma = re.search("\[remote\"origin\"\]url=(.*)\.git", filteredConfig)
```

Comment:
Tried using this with this repo and it gave an invalid URL, could you try to parse the config file to get the origin URL from it?

Reply:
I suggest invoking the git cli instead and reading its output, i.e. `git remote get-url origin`. Additionally, you'll need to convert repos that are accessed via ssh into an HTTP URL.

Reply:
@Snazzah yes, but im thinking about @FichteFoll 's suggestion as well. So, im wondering if in case of an user not having git installed or for some reason git isnt in his system path, we should fallback to the config .git?

Reply:
As a fallback, sure

Reply:
@Snazzah can you take a try to see if it still doesn't work for you?